heap/max_heap.py

```diff
@@ -22,11 +24,9 @@ def get_size(self):
     return len(self.storage)
 
   def _bubble_up(self, index):
```

Comment:
I think this is what we went over together on Python Tutor, right? if so, push the commits so this is updated :)
otherwise, I'm concerned what will happen if you happen to insert a new value that will be the max value. 
also, it seems like this swap only runs once - what if the item needs to bubble up a few times?